Gemfile

```diff
@@ -27,6 +27,15 @@ gem 'jbuilder', '~> 2.5'
 # Use ActiveModel has_secure_password
 # gem 'bcrypt', '~> 3.1.7'
 
+# Bootstrap - source: https://github.com/twbs/bootstrap-rubygem
+gem 'bootstrap', '~> 4.4.1'
```

Comment:
You were able to add gems for Bootstrap, but you still need to add few more code for application.js
Bootstrap needs js
![Screen Shot 2021-04-21 at 9 28 01 PM](https://user-images.githubusercontent.com/35232348/115561816-81e03900-a2e8-11eb-8d1c-4723d98ba6c4.png)

Reply:
Look through your old code and compare it to your Elearning code 🙇